src/pages/Environments.js

```diff
@@ -219,8 +230,8 @@ const Environments = () => {
     ])
   }
   const renderWorkspaceForApps = app => {
-    const { appType, labels: { saturnWorkspaceNamespace, saturnWorkspaceName } } = app
-    const multipleApps = workspaceHasMultipleApps(appsByProject[app.googleProject], appType)
+    const { appType, googleProject, labels: { saturnWorkspaceNamespace, saturnWorkspaceName } } = app
+    const multipleApps = workspaceHasMultipleApps(appsByProject[googleProject], appType)
```

Comment:
Non-functional and unrelated to the bug this PR is aimed to fix. Just noticed that I'd forgotten to pull out `googleProject`.